src/util/logger.ts

```diff
@@ -10,16 +10,47 @@ Object.defineProperty(RegExp.prototype, "toJSON", {
 });
 
 // ===========================================================================
-// TODO: Fix this the next time the file is edited.
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function errJSON(e: any) {
+export function errJSON(e: unknown): Record<string, any> {
   if (e instanceof Error) {
-    return { type: "exception", message: e.message, stack: e.stack };
+    return { type: "exception", message: e.message, stack: e.stack || "" };
+  } else if (typeof e === "object") {
+    return e || {};
   } else {
-    return { message: e.toString() };
+    return { message: (e as object).toString() };
   }
 }
 
+// ===========================================================================
+export const LOG_CONTEXT_TYPES = [
+  "general",
+  "worker",
+  "recorder",
+  "writer",
+  "state",
+  "redis",
+  "storage",
+  "text",
+  "exclusion",
+  "screenshots",
+  "screencast",
+  "originoverride",
```

Comment:
Should we make this `originOverride` to be more consistent with the camel casing used elsewhere in `LOG_CONTEXT_TYPES`?

Reply:
done